src/PrestaShopBundle/Controller/Admin/FrameworkBundleAdminController.php

```diff
@@ -341,6 +341,26 @@ protected function getForbiddenActionMessage($action, $suffix = '')
         throw new \Exception(sprintf('Invalid action (%s)', $action . $suffix));
     }
 
+    /**
+     * Get fallback error message when something unexpected happens
+     *
+     * @param string $type
+     * @param string $code
+     *
+     * @return string
+     */
+    protected function getFallbackErrorMessage($type, $code)
+    {
+        return $this->trans(
+            'An unexpected error occurred. [%type% code %code%]',
```

Comment:
Is _[%type% code %code%]_ part of the translation key?

Reply:
yep, because `code` can be translated.